cmd/create/cluster/cmd.go

```diff
@@ -1917,6 +1943,113 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	isSharedVPC := false
+	privateHostedZoneID := strings.Trim(args.privateHostedZoneID, " \t")
+	privateHostedZoneRoleARN := strings.Trim(args.privateHostedZoneRoleARN, " \t")
+	baseDomain := strings.Trim(args.baseDomain, " \t")
+	if privateHostedZoneID != "" ||
+		privateHostedZoneRoleARN != "" ||
+		baseDomain != "" {
+		if !isSTS {
```

Comment:
We're adding this check to the backend here https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/merge_requests/6333, I think it's ok to remove it from the CLI.
If the feature will be supported for non-STS, we can easily change the backend, unlike the CLI, where we need a release.

Reply:
remove the check